Build failure: xcalib

### Nixpkgs version

- Unstable (25.11)

### Steps to reproduce

build `xcalib`

### Can Hydra reproduce this build failure?

Yes, Hydra can reproduce this build failure.

### Link to Hydra build job

https://hydra.nixos.org/build/309210056

### Relevant log output

```console
> Running phase: configurePhase
       > cmake flags: -GNinja -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTING=OFF -DCMAKE_INSTALL_LOCALEDIR=/nix/store/xdksz3vzb73c6br0im4nwxyhrkw85s56-xcalib-0.11/share/locale -DCMAKE_INSTALL_LIBEXECDIR=/nix/store/xdksz3vzb73c6br0im4nwxyhrkw85s56-xcalib-0.11/libexec -DCMAKE_INSTALL_LIBDIR=/nix/store/xdksz3vzb73c6br0im4nwxyhrkw85s56-xcalib-0.11/lib -DCMAKE_INSTALL_DOCDIR=/nix/store/xdksz3vzb73c6br0im4nwxyhrkw85s56-xcalib-0.11/share/doc/xcalib -DCMAKE_INSTALL_INFODIR=/nix/store/xdksz3vzb73c6br0im4nwxyhrkw85s56-xcalib-0.11/share/info -DCMAKE_INSTALL_MANDIR=/nix/store/xdksz3vzb73c6br0im4nwxyhrkw85s56-xcalib-0.11/share/man -DCMAKE_INSTALL_INCLUDEDIR=/nix/store/xdksz3vzb73c6br0im4nwxyhrkw85s56-xcalib-0.11/include -DCMAKE_INSTALL_SBINDIR=/nix/store/xdksz3vzb73c6br0im4nwxyhrkw85s56-xcalib-0.11/sbin -DCMAKE_INSTALL_BINDIR=/nix/store/xdksz3vzb73c6br0im4nwxyhrkw85s56-xcalib-0.11/bin -DCMAKE_INSTALL_NAME_DIR=/nix/store/xdksz3vzb73c6br0im4nwxyhrkw85s56-xcalib-0.11/lib -DCMAKE_POLICY_DEFAULT_CMP0025=NEW -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_STRIP=/nix/store/dmypp1h4ldn0vfk3fi6yfyf5yxp9yz0k-gcc-wrapper-14.3.0/bin/strip -DCMAKE_RANLIB=/nix/store/dmypp1h4ldn0vfk3fi6yfyf5yxp9yz0k-gcc-wrapper-14.3.0/bin/ranlib -DCMAKE_AR=/nix/store/dmypp1h4ldn0vfk3fi6yfyf5yxp9yz0k-gcc-wrapper-14.3.0/bin/ar -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ -DCMAKE_INSTALL_PREFIX=/nix/store/xdksz3vzb73c6br0im4nwxyhrkw85s56-xcalib-0.11
       > CMake Error at CMakeLists.txt:1 (CMAKE_MINIMUM_REQUIRED):
       >   Compatibility with CMake < 3.5 has been removed from CMake.
       >
       >   Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
       >   to tell CMake that the project requires at least <min> but has been updated
       >   to work with policies introduced by <max> or earlier.
       >
       >   Or, add -DCMAKE_POLICY_VERSION_MINIMUM=3.5 to try configuring anyway.
       >
       > 
       > -- Configuring incomplete, errors occurred!
       For full logs, run:
         nix log /nix/store/bvq2maf02s5rs3y0sq8a4lhwkbm3i832-xcalib-0.11.drv
```

### Additional context

https://github.com/NixOS/nixpkgs/issues/445447

### System metadata

```
$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.12.49, NixOS, 25.11 (Xantusia), 25.11.20251002.7df7ff7`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.28.5`
 - nixpkgs: `/nix/store/9v6qa656sq3xc58vkxslqy646p0ajj61-source`
```

### Notify maintainers



---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

[^1]

[^1]: Please consider [giving up MS GitHub](https://sfconservancy.org/GiveUpGitHub/) or offering a non-proprietary, non-US-corporate-controlled mirror for this free software project. I wish to delete this Microsoft account in the future, but I need more projects like this to support alternative methods to send patches & contribute.

--- a/pkgs/by-name/xc/xcalib/package.nix
+++ b/pkgs/by-name/xc/xcalib/package.nix
@@ -8,6 +8,7 @@
   libXxf86vm,
   libXrandr,
   samurai,
+  fetchpatch,
 }:
 
 stdenv.mkDerivation (finalAttrs: {
@@ -21,6 +22,14 @@ stdenv.mkDerivation (finalAttrs: {
     tag = finalAttrs.version;
     hash = "sha256-o0pizV4Qrb9wfVKVNH2Ifb9tr7N7iveVHQB39WVCl8w=";
   };
+
+  patches = [
+    # bump min cmake to v3.5
+    (fetchpatch {
+      url = "https://codeberg.org/OpenICC/xcalib/commit/e8566ead8c043b5f0003c3613b91deab6430eac8.patch";
+      hash = "sha256-gZc4itfsP5T68ZucdYJWJ4sL11xFaw5ePABsmEYHxrU=";
+    })
+  ];
 
   nativeBuildInputs = [
     cmake

